pkg/scheduler/periodic.go

```diff
@@ -150,7 +150,7 @@ func (s *PeriodicScheduler) generateName() string {
 	if s.getTargetNamespace() != invMeta.Namespace {
 		return meta_util.ValidCronJobNameWithPrefixNSuffix(apis.PrefixStashTrigger, invMeta.Namespace, strings.ReplaceAll(invMeta.Name, ".", "-"))
 	}
-	return meta_util.ValidCronJobNameWithPrefixNSuffix(apis.PrefixStashTrigger, "", strings.ReplaceAll(invMeta.Name, ".", "-"))
+	return meta_util.ValidCronJobNameWithSuffix(apis.PrefixStashTrigger, strings.ReplaceAll(invMeta.Name, ".", "-"))
```

Comment:
I think we need to also delete the cronjobs that were create by the bad name format.